.github/workflows/deploy-vercel-staging.yml

```diff
@@ -254,8 +177,8 @@ jobs:
         with:
           variables: |
             CUSTOMER_REF_TO_DEPLOY=${{ env.CUSTOMER_REF_TO_DEPLOY }}
-            VERCEL_DEPLOYMENT_URL=${{ env.VERCEL_DEPLOYMENT_URL }}
             VERCEL_DEPLOYMENT_DOMAIN=${{ env.VERCEL_DEPLOYMENT_DOMAIN }}
+            VERCEL_DEPLOYMENT_URL=${{ format('https://{0}', env.VERCEL_DEPLOYMENT_DOMAIN) }}
```

Comment:
Shouldn't the VERCEL_DEPLOYMENT_URL and VERCEL_DEPLOYMENT_DOMAIN both be returned by the Action?

Reply:
That'd avoid doing the above which isn't dev-friendly

Reply:
Yes I agree